.travis.yml

```diff
@@ -8,11 +8,15 @@ rvm:
   - 2.2
   - 2.3.3
   - 2.4.0
+  - ruby-head
 
 matrix:
   include:
     - rvm: 2.3.0
       script: bundle exec rake test_isolated
+  allow_failures:
+    - rvm: ruby-head
+  fast_finish: true
```

Comment:
I don't think this is needed since we don't have in Rails and the results are sent as soon the default group finish.